cmd/samAssembler/prior.go

```diff
@@ -0,0 +1,197 @@
+package main
```

Comment:
Note to self: this file (cmd/samAssembler/prior.go) seems like the main new functional code